packages/pages/src/vite-plugin/build/closeBundle/functionMetadata.ts

```diff
@@ -1,21 +1,21 @@
-import fs from "fs";
-import path from "path";
-import esbuild from "esbuild";
-import { importFromString } from "module-from-string";
-import glob from "glob";
+import fs from "fs"
+import path from "path"
+import esbuild from "esbuild"
+import { importFromString } from "module-from-string"
+import glob from "glob"
 
-const FUNCTIONS_PATH = path.resolve("functions");
+const FUNCTIONS_PATH = path.resolve("functions")
 const FUNCTION_METADATA_PATH = path.join(
   FUNCTIONS_PATH,
   "functionMetadata.json"
-);
-const TEMP_DIR = ".temp";
+)
+const TEMP_DIR = ".temp"
```

Comment:
Little scope creep-ey but can we use os.tmpdir here https://nodejs.org/api/os.html#ostmpdir